nasa-jpl/explorer-1
# Adds semantic colors and base styles for form elements

### Checklist

  - PR titles are used to generate the change log in [releases](../releases); good ones make that easier to scan.
  - PRs will be broadly categorized in the change log, but for even easier scanning, consider prefixing with a component name or other useful categorization, e.g., "BaseButton: fix layout bug", or "Storybook: Update dependencies".
  - These tags are used to do the aforementioned broad categorization in the change log and determine what the next release's version number should be.
  - [Release Drafter](https://github.com/marketplace/actions/release-drafter) will attempt to do the category labeling for you! Please double-check its work.

## Description

Fixes #42:
- Updates @tailwindcss/forms to v0.3.4
- Adds all semantic colors to branding guidelines
- Adds `/src/scss/_forms.scss` as base styles for form elements. 
- Adds a few Forms stories to illustrate the updated styles
- Updates `explorer-1.min.css`

## Breaking changes
- Input fields will now default to having a `gray-light-mid` border. For input fields that should not have a border, add class `border-0` to them.
- Projects importing SCSS partials a la carte will need to import the new `_forms.scss` partial as one of the first imports in the main SCSS file.
- Projects that have defined their own semantic colors should remove those color definitions from their TailwindCSS config and use those provided by Explorer-1 instead.

## Instructions to test

1. `npm run storybook`
2. Check the stories under Components/Forms to see these styles in action.
3. Check the [Colors](http://localhost:6006/?path=/story/brand-identity-colors--page) story to see all semantic colors added.

## Tested in the following environments/browsers:

### Operating System


### Browser
